app/src/main/res/layout/fragment_user_profile.xml

```diff
@@ -0,0 +1,141 @@
+        android:layout_height="55dp"
+        android:layout_marginLeft="35dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginRight="35dp"
+        android:gravity="left|center_vertical"
+        android:text="@string/assistance_button"
+        android:textColor="@color/black"
+        app:backgroundTint="@color/white"
+        app:fontFamily="@font/inter_semibold"
+        app:icon="@drawable/help"
+        app:iconPadding="15dp"
+        app:iconTint="@color/black"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/settings_button"
+        app:strokeColor="@color/black"
+        app:strokeWidth="1dp" />
+
+    <Button
+        android:id="@+id/legal_terms_button"
+        android:layout_width="match_parent"
+        android:layout_height="55dp"
+        android:layout_marginLeft="35dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginRight="35dp"
+        android:gravity="left|center_vertical"
+        android:text="@string/legal_terms_button"
+        android:textColor="@color/black"
+        app:backgroundTint="@color/white"
+        app:fontFamily="@font/inter_semibold"
+        app:icon="@drawable/legal_terms"
+        app:iconPadding="15dp"
+        app:iconTint="@color/black"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/assistance_button"
+        app:strokeColor="@color/black"
+        app:strokeWidth="1dp" />
+
+    <Button
+        android:id="@+id/completed_activities_button"
+        android:layout_width="match_parent"
+        android:layout_height="120dp"
+        android:layout_marginLeft="35dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginRight="35dp"
+        android:gravity="left|center_vertical"
+        android:text="@string/completed_activities_button"
+        app:backgroundTint="@color/seed"
+        app:cornerRadius="15dp"
+        app:fontFamily="@font/inter_semibold"
+        app:icon="@drawable/done"
+        app:iconPadding="15dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/legal_terms_button" />
+
+    <Button
+        android:id="@+id/log_out_button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="32dp"
+        android:text="@string/log_out_button"
+        app:fontFamily="@font/inter_semibold"
+        app:backgroundTint="@color/black"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.498"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/completed_activities_button" />
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
A newline at the end of the file is [considered good practice](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).